TramsDataApi/Gateways/CensusDataGateway.cs

```diff
@@ -0,0 +1,28 @@
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using CsvHelper;
+using TramsDataApi.CensusData;
+using System;
+
+
+namespace TramsDataApi.Gateways
+{
+    public class CensusDataGateway : ICensusDataGateway
+    {
+        
+        public CensusDataModel GetCensusDataByURN(string urn)
+        {
+            string workingDirectory = Environment.CurrentDirectory;
+            string projectDirectory = Directory.GetParent(workingDirectory).Parent.Parent.Parent.FullName;
+            
+            
+            using (var reader = new StreamReader($"{projectDirectory}/TramsDataApi/CensusData/2018-2019_england_census.csv"))
+            using (var csv = new CsvReader(reader, CultureInfo.InvariantCulture))
+            {
+                var records = csv.GetRecords<CensusDataModel>();
+                return records.FirstOrDefault(censusData => censusData.URN == urn);
```

Comment:
Not a comment on this PR specifically, but we should probably make a note to track the performance of this - loading a ~24k line CSV into memory on each call will (presumably) be very slow, so we'll need to make a note to refactor this (to a singleton?) when we see it's got a big impact